<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.176136 rgba(91, 156, 129, 255), stop:0.943182 rgba(135, 207, 227, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Sucursal Veterinaria</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Veterinario a Cargo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha consulta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Tratamientos consulta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Medicamentos consulta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Vacunas suministradas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Causa de la visita</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Frecuencia respiratoria</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Frecuencia cardiaca</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>600</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Peso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSucursal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputVetCargo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputCausaVisita">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFrecRespiratoria">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFrecCardiaca">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPeso">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>660</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputEdad">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>660</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttAgregarTratamientos">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttAgregarMedicamentos">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="tratamientosList">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="medicamentosList">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="fechaConsulta">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="vacunasList">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttAgregarVacunas">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>720</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTemp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>720</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAgregarFicha">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>790</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Agregar Ficha general</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAgregarFichaHosp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>790</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>false</italic>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Add Ficha Hospitalizacion</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAgregarFichaSedacion">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>790</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Ficha Sedación</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAgregarFichaOperacion">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>790</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Ficha Operación</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>inputSucursal</zorder>
   <zorder>inputVetCargo</zorder>
   <zorder>inputCausaVisita</zorder>
   <zorder>inputFrecRespiratoria</zorder>
   <zorder>inputFrecCardiaca</zorder>
   <zorder>inputPeso</zorder>
   <zorder>label_11</zorder>
   <zorder>inputEdad</zorder>
   <zorder>buttAgregarTratamientos</zorder>
   <zorder>buttAgregarMedicamentos</zorder>
   <zorder>tratamientosList</zorder>
   <zorder>medicamentosList</zorder>
   <zorder>fechaConsulta</zorder>
   <zorder>vacunasList</zorder>
   <zorder>buttAgregarVacunas</zorder>
   <zorder>label_12</zorder>
   <zorder>inputTemp</zorder>
   <zorder>botonAgregarFicha</zorder>
   <zorder>botonAgregarFichaSedacion</zorder>
   <zorder>botonAgregarFichaOperacion</zorder>
   <zorder>botonAgregarFichaHosp</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
